app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -63,19 +63,49 @@ class PipelineSampleReport extends React.Component {
     return params.selected_genus != 'None' && params.disable_filters != 1;
   }
 
+  //path to NCBI 
+  gotoNCBI(e) {
+    const taxId = e.target.getAttribute('data-tax-id');
+    const ncbiLink = `https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=${taxId}`;
+    window.open(ncbiLink, '_blank');
+  }
+
+  //download Fasta  
+  downloadFastaUrl(e) {
+    const pipelineId = e.target.getAttribute('data-pipeline-id');
+    const taxLevel = e.target.getAttribute('data-tax-level');
+    const taxId = e.target.getAttribute('data-tax-id');
+    console.log(pipelineId, taxLevel, taxId, 'fasta params');
+    location.href = `/pipeline_outputs/${pipelineId}/fasta/${taxLevel}/${taxId}/NT_or_NR`;
+  }
+
+  displayTags(taxInfo, reportDetails) {
+    return (
+      <span className="link-tag">
+        { taxInfo.tax_id > 0 ? <i data-tax-id={taxInfo.tax_id} onClick={this.gotoNCBI} className="fa fa-link cloud" aria-hidden="true"></i> : null }
+        { taxInfo.tax_id > 0 && reportDetails.taxon_fasta_flag ? <i data-pipeline-id={reportDetails.pipeline_info.id} 
```

Comment:
you might want to check with @cdebourcy on if the download link should be displayed when tax_id is less than 0.

Reply:
yes, link should be displayed even when tax_id < 0 -- that way, the user can inspect these "uncategorized" reads